modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -83,10 +87,24 @@ protected function register_controls() {
 		$end = is_rtl() ? 'left' : 'right';
 		$start_logical = is_rtl() ? 'end' : 'start';
 		$end_logical = is_rtl() ? 'start' : 'end';
-		$heading_selector_non_touch_device = '{{WRAPPER}}.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs[data-touch-mode="false"] > .e-n-tabs-heading';
-		$heading_selector_touch_device = '{{WRAPPER}}.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs[data-touch-mode="true"] > .e-n-tabs-heading';
-		$heading_selector = '{{WRAPPER}}.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading';
-		$content_selector = ':where( {{WRAPPER}}.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content ) > .e-con';
+
+		$optimized_markup = Plugin::$instance->experiments->is_feature_active( 'e_optimized_markup' ) && ! $this->has_widget_container();
+
+		$heading_selector_non_touch_device = $optimized_markup
```

Comment:
@louiswol94 

Could we implement it like this: 

$container_selector = $optimized_markup ? '' : '.elementor-widget-container > ';
$heading_selector_non_touch_device = "{{WRAPPER}}.elementor-widget-n-tabs > {$container_selector}.e-n-tabs[data-touch-mode=\"false\"] > .e-n-tabs-heading";

This is will create less duplication in the code. 

Wdyt?